adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -269,7 +273,11 @@ - (nullable NSError *)loadAd:(nonnull NSString *)placement
   if ([self isAdCachedForPlacementID:placement withDelegate:delegate]) {
     [delegate adAvailable];
   } else {
-    NSError *loadError;
+    if ([_prioritizedPlacementID isEqualToString:placement]) {
+      return nil;
+    }
```

Comment:
Can we add a comment why we are returning nil if placement is same as previous placement?

Reply:
@shiv-vungle Got it, I will add a comment to explain why we return nil in this case. Thanks a lot for the reminder. ^_^

Reply:
@shiv-vungle Added the comment to explain why we return nil in this case in `6.7.x_pre_release` branch. Thanks a lot. ^_^